### ISSUE TITLE
Notifications screen

### ISSUE THREAD

**Issue by anotherwebguy** [infiniteoverflow/Libro#116]
I would like to create the notifications screen layout for the app plz assign me this issue under gssoc @infiniteoverflow

**Comment by anotherwebguy** [comment]
> I would like to create the notifications screen layout for the app plz assign me this issue under gssoc @infiniteoverflow

hey plz assign me this issue @infiniteoverflow  @Souvikns @pratyushmp

**Comment by infiniteoverflow** [comment]
Yes , @anotherwebguy You can get started with this. First create a notification template and then run it over using ListView.builder()
Once done, just confirm the ui with us and then later proceed with the PR.

**Comment by anotherwebguy** [comment]
> Yes , @anotherwebguy You can get started with this. First create a notification template and then run it over using ListView.builder()
> Once done, just confirm the ui with us and then later proceed with the PR.

Yeah sure @infiniteoverflow

**Comment by anotherwebguy** [comment]
![image](https://user-images.githubusercontent.com/66346161/116869997-5a199b00-ac2f-11eb-8509-c6685fc0d866.png)
![image](https://user-images.githubusercontent.com/66346161/116870005-5dad2200-ac2f-11eb-9e66-cc3ff6ee1921.png)

hello @infiniteoverflow  i'am done with the template layout 
should i make the pr now?

**Comment by infiniteoverflow** [comment]
Yes , @anotherwebguy You can go ahead. It looks good.
